sim/common/sod/items_sets/phase_5.go

```diff
@@ -0,0 +1,69 @@
+package item_sets
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+)
+
+///////////////////////////////////////////////////////////////////////////
+//                                 Cloth
+///////////////////////////////////////////////////////////////////////////
+
+
+///////////////////////////////////////////////////////////////////////////
+//                                 Leather
+///////////////////////////////////////////////////////////////////////////
+
+
+///////////////////////////////////////////////////////////////////////////
+//                                 Mail
+///////////////////////////////////////////////////////////////////////////
+
+
+///////////////////////////////////////////////////////////////////////////
+//                                 Plate
+///////////////////////////////////////////////////////////////////////////
+
+
+///////////////////////////////////////////////////////////////////////////
+//                                 Other
+///////////////////////////////////////////////////////////////////////////
+
+var ItemSetTwinBladesofHakkari = core.NewItemSet(core.ItemSet{
+	Name: "The Twin Blades of Hakkari",
+	Bonuses: map[int32]core.ApplyEffect{
+		// Increases Swords +3
+		// 2% chance on melee hit to gain 1 extra attack.  (1%, 100ms cooldown)
+		2: func(agent core.Agent) {
+			character := agent.GetCharacter()
+			character.PseudoStats.SwordsSkill += 3
+			if !character.AutoAttacks.AutoSwingMelee {
+				return
+			}
+	
+			icd := core.Cooldown{
+				Timer:    character.NewTimer(),
+				Duration: time.Millisecond * 100,
+			}
+	
+			character.GetOrRegisterAura(core.Aura{
+				Label:    "Twin Blades of the Hakkari",
+				Duration: core.NeverExpires,
+				OnReset: func(aura *core.Aura, sim *core.Simulation) {
+					aura.Activate(sim)
+				},
```

Comment:
🔨 little knit: Trying to use helpers more so this could be
```suggestion
			core.MakePermanent(character.GetOrRegisterAura(core.Aura{
				Label:    "Twin Blades of the Hakkari",
				...
		  }))
```